src/util/Cache.h

```diff
@@ -476,16 +469,16 @@ class HeapBasedLRUCache
                      detail::timeUpdater, detail::timeAsScore, ValueSizeGetter>;
 
  public:
-  explicit HeapBasedLRUCache(size_t capacityNumEls = size_t_max,
-                             size_t capacitySize = size_t_max,
-                             size_t maxSizeSingleEl = size_t_max)
+  explicit HeapBasedLRUCache(
+      size_t capacityNumEls = size_t_max,
+      MemorySize capacitySize = MemorySize::bytes(size_t_max),
+      MemorySize maxSizeSingleEl = MemorySize::bytes(size_t_max))
```

Comment:
Maybe implement `MemorySize::max()` for this case to not leak the implementation detail that its 64 bytes here.

Reply:
Alright. Started a PR for this: https://github.com/ad-freiburg/qlever/pull/1113

Reply:
Done, see [dd74fdd](https://github.com/ad-freiburg/qlever/pull/1111/commits/dd74fdd94b23b4bc3bea8fa7aa4f8e60f741727f).